internal/bytesconv/bytesconv_1.19.go

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a MIT style
 // license that can be found in the LICENSE file.
 
+//go:build !go1.20
+// +build !go1.20
```

Comment:
please remove `+build !go1.20`, because we support min go version = 1.18

Reply:
Done.